app/(gcforms)/[locale]/(form administration)/form-builder/[id]/preview/Preview.tsx

```diff
@@ -88,31 +90,18 @@ export const Preview = ({
 
   const brand = formRecord?.form ? formRecord.form.brand : null;
 
-  const isPastClosingDate = useIsFormClosed();
-
   const hasHydrated = useRehydrate();
 
   const isShowReviewPage = showReviewPage(formRecord.form);
 
   if (isPastClosingDate) {
```

Comment:
We shouldn't end up here as this is checked at the page level -- but added meta redirect just in case to do a server re-load.